anaconda.py

```diff
@@ -48,7 +48,13 @@ def exitHandler(rebootData):
     # pylint: disable=used-before-assignment
     if "nokill" in kernel_arguments:
         util.vtActivate(1)
-        print("anaconda halting due to nokill flag.")
+        print("Warning - 'nokill' kernel option detected")
```

Comment:
I'm thinking if this shouldn't go to stderr instead but given the fact that the current output is not going there I'm honestly not sure.

Reply:
Looks like we are using the `stdoutLog` quite a log in `anaconda.py`: https://github.com/rhinstaller/anaconda/blob/main/anaconda.py#L236

That would be ideal, as the output goes to standard output *and* is also logged (so should end up in Journal as well, for possible later analysis). The only issue could be if the logging machinery is initialized at this early point. :P

Reply:
Changed to `log.info` :)